clinica/iotools/bids_utils.py

```diff
@@ -485,25 +493,23 @@ def write_scans_tsv(bids_dir, bids_ids, scans_dict):
     import os
     from glob import glob
     from os import path
-
     import pandas as pd
 
     for bids_id in bids_ids:
-        # Create the file
         sessions_paths = glob(path.join(bids_dir, bids_id, "ses-*"))
         for session_path in sessions_paths:
-            scans_df = pd.DataFrame()
             session_name = session_path.split(os.sep)[-1]
-            tsv_name = bids_id + "_" + session_name + "_scans.tsv"
-
+            scans_df = pd.DataFrame()
```

Comment:
More of a nitpick, but please avoid moving code around without touching it 🙏 

The lighter the diff of the PR, the easier it gets for people to review your work

Reply:
My bad. Automatic merge problem: I was coding this feature and it was coded elsewhere.